lib/matplotlib/tests/test_image.py

```diff
@@ -922,3 +922,20 @@ def test_relim():
     ax.relim()
     ax.autoscale()
     assert ax.get_xlim() == ax.get_ylim() == (0, 1)
+
+def test_edgefacecolor_pcolor():
+    # Test that pcolor/pcolormesh honours edgecolors kwarg when facecolors
+    # is set 'none' (github issue #1302 and fixed by PR #12226)
+    fig, ax = plt.subplots(2)
+    im0 = ax[0].pcolor(np.arange(12).reshape(4, 3),
+                       edgecolors='red', facecolors='none')
+    im1 = ax[1].pcolormesh(np.arange(12).reshape(4, 3),
+                       edgecolors='red', facecolors='none')
+    buff = io.BytesIO()
+    # The figure needs to be saved in order to trigger draw() i.e.
+    # to make sure that the edges of the mesh are drawn.
+    fig.savefig(buff, format='png')
```

Comment:
You can just do `fig.canvas.draw()` I think...

Reply:
you are right. pushed a change (also squashed the commits).